src/routes/login/+page.svelte

```diff
@@ -33,45 +39,15 @@
 		<p class="error">You have entered the wrong credentials.</p>
 	{/if}
 
-	<button type="submit">Login</button>
+	<Button label="Log In" button_type="submit" type="confirm" onclick="{() => null}"/>
 </form>
 
-<div class="actions">
-	No Account? Sign up <a href="/signup">here</a>!<br />
-	<a class="btn" href="/">Back</a>
-</div>
+<Divider />
+
+<LinkButton destination="/signup" label="Signup here!"/>
+<LinkButton destination="/" label="Back"/>
```

Comment:
Can you put these in the same line? There is no real reason each of them should occupy a whole line